### DISCUSSION TITLE
Notifica ANSC in ANPR solo dopo la Conferma delle Notifiche di Annotazione automatica.

### DISCUSSION THREAD

**Discussion by gcuci** [italia/ansc#969] [Miglioramenti] [answered]
Buongiorno,
 
segnaliamo che dopo aver validato e firmato un evento di morte, ed inviata la data di validità con l'apposito servizio (avendo l'evento una risultanza anagrafica), riceviamo la Notifica di Trascrizione, e quella di Annotazione automatica per la nascita.
 
Confermando la Notifica di Trascrizione, non troviamo però nell'ANPR di Test del Comune (tramite Web App), la relativa Notifica ANSC, che invece risulta disponibile, subito dopo la conferma anche dell'Annotazione automatica. Abbiamo verificato che il comportamento è simile anche per le altre tipologie di evento. La Notifica ANSC è presente in ANPR, solo dopo aver confermato, oltre alla Notifica di trascrizione, anche quelle eventuali di Annotazione automatica.
 
Pensiamo non sia corretto questo comportamento, e che la Notifica in ANPR, dovrebbe già essere disponibile dopo la Conferma della Notifica di Trascrizione.
 
In attesa di riscontro,
 
cordiali saluti.

**Comment by asqueglia** [comment, marked as answer]
Buongiorno, il comportamento è stato voluto e richiesto dal Ministero, come più volte detto durante le call. Si prega di rivolgersi al Dipartimento per segnalare le vostre osservazioni al fine di poter rivedere il flusso implementato.
